test/gracefulstop_test.go

```diff
@@ -122,7 +122,8 @@ func (s) TestGracefulStop(t *testing.T) {
 		},
 	}
 	s := grpc.NewServer()
-	testgrpc.RegisterTestServiceServer(s, ss)
+	ss.S = s
```

Comment:
i think you have to assign the listener as well when creating the stub server?

Reply:
Done